src/_trio_check_attrs_aliases.py

```diff
@@ -0,0 +1,22 @@
+"""Conftest is executed by Pytest before test modules.
+
+We use this to monkeypatch attrs.field(), so that we can detect if aliases are used for test_exports.
+"""
+
+from typing import Any
+
+import attrs
+
+orig_field = attrs.field
+
+
+def field(**kwargs: Any) -> Any:
+    original_args = kwargs.copy()
+    metadata = kwargs.setdefault("metadata", {})
+    metadata["original_args"] = original_args
```

Comment:
Probably should add `trio` to this key somewhere, since it's possible one of our dependencies might use `original_args` in attrs metadata, or whatever. Not sure why you're storing all the args, if we only use `alias`? Future-proofing?

Reply:
Yeah, for any sort of later attrs test

Reply:
Would suggest using `object` if possible